docs/reference/commandline/dockerd.md

```diff
@@ -1172,6 +1173,9 @@ set this parameter separately for each daemon.
 - `-p, --pidfile=/var/run/docker.pid` is the path where the process ID of the daemon is stored. Specify the path for your
 pid file here.
 - `--host=[]` specifies where the Docker daemon will listen for client connections. If unspecified, it defaults to `/var/run/docker.sock`.
+- `--iptables=false` prevents the Docker daemon from adding iptables rules. If
+  multiple daemons manage iptables rules, they may overwrite rules set by
+  another daemon
```

Comment:
Add a period at the end of the sentence.

It's probably important to note that this limits networking choices to `--net=none` or `--net=host`.